components/brave_wallet_ui/constants/types.ts

```diff
@@ -421,13 +416,16 @@ export interface PortfolioTokenHistoryAndInfo {
 
 interface BaseTransactionParams {
   network: BraveWallet.NetworkInfo
-  fromAccount: WalletAccountType
+  fromAddress: string
```

Comment:
We need to stick with `WalletAccountType` here. This works only for account-based blockchains, and has been refactored recently.

The mojo interfaces for creating unapproved transactions will soon reflect the same.

Reply:
For now, the `BraveWallet.AccountInfo` object.

Once #18564 is merged, we'll have an `accountId` field in `BraveWallet.AccountInfo`, so we'll be able to pass that instead.

<img width="888" alt="Screenshot 2023-05-25 at 21 40 17" src="https://github.com/brave/brave-core/assets/3684187/605e4a46-1516-498f-863a-999b8a3f2746">

Reply:
updated to use a subset of `WalletAccountType`